CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## Unreleased
+
+### Dependencies
+
+- Bump Sentry iOS SDK to `2.27.1` ([#231](https://github.com/getsentry/sentry-capacitor/pull/231))
```

Comment:
typo: should be `7.27.1`

Reply:
also I think it would be better to keep referring to the dependency as `sentry-cocoa`, so as not to confuse ppl who are not aware that this is the iOS SDK